mismatched types expected i8, found u8 targeting i686-linux-android

```
error[E0308]: mismatched types
  --> /home/jarod/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.0/src/dylib.rs:30:32
   |
30 |         let ptr = libc::dlopen(name.as_ptr(), libc::RTLD_LAZY);
   |                                ^^^^^^^^^^^^^ expected i8, found u8
   |
   = help: here are some functions which might fulfill your needs:
           - .offset(...)
           - .wrapping_offset(...)

```

--- a/src/dylib.rs
+++ b/src/dylib.rs
@@ -3,7 +3,7 @@ use std::marker;
 use std::mem;
 use std::sync::atomic::{AtomicUsize, Ordering};
 
-use libc::{self, c_void};
+use libc::{self, c_char, c_void};
 
 pub struct Dylib {
     pub init: AtomicUsize,
@@ -27,7 +27,7 @@ impl Dylib {
             return true
         }
         let name = CString::new(path).unwrap();
-        let ptr = libc::dlopen(name.as_ptr(), libc::RTLD_LAZY);
+        let ptr = libc::dlopen(name.as_ptr() as *const c_char, libc::RTLD_LAZY);
         if ptr.is_null() {
             return false
         }

